neptun/src/device/integration_tests/mod.rs

```diff
@@ -477,6 +473,33 @@ mod tests {
         assert!(response.ends_with("errno=0\n\n"));
     }
 
+    #[test]
+    #[ignore]
```

Comment:
Is it right that we don't want to normally run these tests?

Reply:
That's just the way Boringtun used to differentiate integration tests. They are marked as ignored and you run them via `cargo tests --ignored`